src/components/SearchBoxMobile.js

```diff
@@ -0,0 +1,29 @@
+import SearchIcon from '@mui/icons-material/Search';
+import { Box, IconButton, InputBase } from '@mui/material';
+import React from 'react';
+
+export default function SearchBox() {
+  return (
+    <Box>
+      <InputBase
+        sx={{
+          height: '44px',
+          width: '356px',
+          borderRadius: '50px',
+          paddingLeft: '20px',
+          background: '#FFFFFF',
+          boxShadow: '0px 4px 10px rgba(0,0,0,0.25)',
+          fontSize: 'small',
```

Comment:
```suggestion
          fontSize: 'small',
          fontStyle: 'italic',
```

Reply:
I have made the change. See pic below.

![image](https://user-images.githubusercontent.com/61099792/148627277-32f7c2d2-6e49-4fdb-83e0-db652c57b8b9.png)